app/services/fedora_checker.rb

```diff
@@ -0,0 +1,130 @@
+require "byebug"
+
+class FedoraChecker
+  def initialize(options:)
+    @options = options
+    File.open(options[:metadata_path], 'r') do |f|
+      @data = Yajl::Parser.new.parse(f)
```

Comment:
Let's try just `JSON.parse` without the extra dependency, since we aren't using streaming. If it gives you any trouble don't worry about it just use yajl, the Gemfile comment could be even more explicit that remove this when we're done with fedora.

Reply:
No evidence that yajl is faster than plain old JSON.parse. Getting rid of yajl.